app/views/portal/show.rb

```diff
@@ -0,0 +1,871 @@
+module Portal
+  class Show < ApplicationView
+    attr_accessor :document, :debug
+
+    def page_title
+      [@document.fetch(:title, ['']).join(', '), 'Europeana'].compact.join(' - ')
+    end
+
+    def navigation
```

Comment:
Cyclomatic complexity for navigation is too high. [8/6]<br>Assignment Branch Condition size for navigation is too high. [48.76/15]<br>Method has too many lines. [43/10]<br>Perceived complexity for navigation is too high. [9/7]